packages/app-desktop/gui/MainScreen/commands/notesSortOrderToggleReverse.ts

```diff
@@ -0,0 +1,18 @@
+import CommandService, { CommandContext, CommandDeclaration, CommandRuntime } from '@joplin/lib/services/CommandService';
+import Setting from '@joplin/lib/models/Setting';
+import { _ } from '@joplin/lib/locale';
+
+export const declaration: CommandDeclaration = {
+	name: 'notesSortOrderToggleReverse',
+	label: () => Setting.settingMetadata('notes.sortOrder.reverse').label(),
```

Comment:
That's sensible, but we shouldn't actually access Settings from the declaration. They are probably initialised by then we there's no guarantee. So you'll just need to copy and paste the string.

Reply:
I see.

Reply:
Finished. 36e00f8